weasyl/controllers/interaction.py

```diff
@@ -71,26 +70,25 @@ def frienduser_(request):
 @login_required
 @token_checked
 def unfrienduser_(request):
-    form = request.web_input(userid="", feature="")
-    otherid = define.get_int(form.userid)
+    otherid = define.get_int(request.params.get('userid', ''))
 
     if request.userid == otherid:
         raise WeasylError('cannotSelfFriend')
 
     frienduser.remove(request.userid, otherid)
 
-    raise HTTPSeeOther(location="/manage/friends?feature=%s" % form.feature)
+    raise HTTPSeeOther(location="/manage/friends?feature=%s" % request.params.get('feature'))
```

Comment:
Don’t want `None` showing up in the URL. For the original behaviour,

```python
raise HTTPSeeOther(location="/manage/friends?feature=%s" % request.params.get('feature', ''))
```

but I think this is better:

```suggestion
    redirect = "/manage/friends"
    
    if request.params.get('feature') == 'pending':
        redirect += "?feature=pending"

    raise HTTPSeeOther(location=redirect)
```